socorro/external/postgresql/adi.py

```diff
@@ -63,7 +63,7 @@ def get(self, **kwargs):
 
         sql = """
             SELECT
-                SUM(adu_count) AS adi_count,
+                SUM(adu_count)::BIGINT AS adi_count,
```

Comment:
If we don't do this, the `adi_count` summation will become a `decimal.Decimal` object when piped via pyscopg2.